src/sentry/static/sentry/app/views/dashboardsV2/styles.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import {Widget} from './types';
+
+export const WidgetWrapper = styled(motion.div)<{displayType: Widget['displayType']}>`
```

Comment:
will we have more styles in here? otherwise, I would convert it to  a `WidgetWrapper` component

Reply:
ok will do